packages/examples/core/primitives/sphere.js

```diff
@@ -0,0 +1,29 @@
+/*
+// title       : Sphere
+// author      : Rene K. Mueller
+// license     : MIT License
+// description : testing sphere() and geodesicSphere() functions
+// file        : sphere.js
+// tags        : sphere, geodesic, geodesicsphere, ellipsoid, shape, 3d
+*/
+
+const { sphere, geodesicSphere } = require('@jscad/modeling').primitives
+const { translate, scale } = require('@jscad/modeling').transforms
+
+// TODO - Test with new transform apply orders
```

Comment:
it seems that transforms are being used... is this done?

Reply:
Yep, looks good.